### PR TITLE
Add double data type

### LINKED ISSUE #498: Add double column type
MySQL has a `double` column type. Please add it.

### PR DESCRIPTION
**Pull request by ntzm**
Closes #498

### CODE REVIEW

File: src/Phinx/Db/Adapter/MysqlAdapter.php [hunk 1]
```diff
@@ -41,7 +41,7 @@
 class MysqlAdapter extends PdoAdapter implements AdapterInterface
 {
 
-    protected $signedColumnTypes = ['integer' => true, 'biginteger' => true, 'float' => true, 'decimal' => true, 'boolean' => true];
+    protected $signedColumnTypes = ['integer' => true, 'biginteger' => true, 'float' => true, 'decimal' => true, 'double' => true, 'boolean' => true];
```
[the remark this answers is not in the document]
Comment (ntzm) [reply src/Phinx/Db/Adapter/MysqlAdapter.php]: Should I wrap this to one item per line?

**Comment by ntzm** [comment]
This is basically a copy of #1069 but with the signed column

File: src/Phinx/Db/Adapter/MysqlAdapter.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (dereuromark) [reply src/Phinx/Db/Adapter/MysqlAdapter.php]: Would probably be good, yeah.

**Comment by dereuromark** [comment]
Do we want to add any (regression) tests for this new feature?

**Comment by ntzm** [comment]
Sure, give me a sec

**Comment by ntzm** [comment]
Gonna have to leave this till tomorrow unfortunately, should have updated by 1:30 BST tomorrow

**Comment by ntzm** [comment]
How's that? Sorry if I haven't done anything properly.

Also, what should be done about SQL Server? Don't think they support doubles

**Comment by chinpei215** [comment]
Sorry for the late reply. I think you could use `double precision` or `float` for SQL Server. See:
https://docs.microsoft.com/en-us/sql/t-sql/data-types/float-and-real-transact-sql
> The synonym for double precision is float(53).

**Comment by dereuromark** [comment]
@chinpei215 Should we continue here?

**Comment by piotr-cz** [comment]
What happened with this PR?

**Comment by lorenzo** [comment]
@piotr-cz there are conflicts with the master branch. If you want to continue it, please open a new PR. I'll close it as it is dormant now, and cannot be merged.

**Pull request closed without merging**